samples/metaapp/src/main/java/com/sourcepointmeta/metaapp/tv/demo/DemoActivityTv.kt

```diff
@@ -83,6 +83,13 @@ class DemoActivityTv : FragmentActivity() {
         }
     }
 
+    override fun onBackPressed() {
+        if (spConsentLib.isWebviewShown().not())
+            super.onBackPressed()
+        else
+            spConsentLib.onBackPressed()
+    }
```

Comment:
@Nevazhnovu does that mean the application has to implement that? Since we own the view and we have access to the activity itself, can't the SDK override the `onBackPressed` and have it transparent to the host app?

Reply:
@andresilveirah but we also don't want to prohibit the user from being able to utilize their application's `onBackPressed()`

meaning that:
• WHEN there are no `webView` on the screen
• AND the user clicks back button (maybe to go out from the screen were our `webView` was triggered)
• THEN the user should be redirected from that screen

and also we should be careful about this case:
• WHEN there is a visible `webView` on the screen
• AND the user clicks back button and expects to go back inside the `webView`
• THEN the `onBackPressed()` should _ONLY_ be triggered for the `webView` and not for the entire activity

Reply:
@andresilveirah AFAIK in Android only Activity/Context which is `DemoActivityTv` can overload the `onBackPressed()` method.  @bohdan-go-wombat Proof me wrong if I am